src/modules/pages/components/RouteLayouts/UserNavigation.tsx

```diff
@@ -180,17 +184,19 @@ const UserNavigation = () => {
         )}
         {previousWalletConnected && attemptingAutoLogin && userDataLoading ? (
           <div className={styles.walletAutoLogin}>
-            <MiniSpinnerLoader title={MSG.walletAutologin} />
+            <MiniSpinnerLoader
+              title={MSG.walletAutologin}
+              titleTextValues={{ isMobile: false }}
+            />
           </div>
         ) : (
           <div className={`${styles.elementWrapper} ${styles.walletWrapper}`}>
             {userCanNavigate && nativeToken && userLock && (
               <UserTokenActivationButton
-                nativeToken={nativeToken}
-                userLock={userLock}
+                userTokenData={getUserTokenData(userLock)}
```

Comment:
I've extracted the logic for getting user token data into a separate function which is being shared between this component and also `AvatarDropdownPopoverMobile`